radar.csv

```diff
@@ -153,3 +153,5 @@ Physical whiteboards,adopt,tools,false,"Some teams use physical whiteboards for
 "Github Pages",explore,tools,true,"<a href='https://pages.github.com/'>Github Pages</a> is a lightweight website platform, hosted from a Github repo. Support for branching/merging may make this a useful tool for internal documentation. The coaches are exploring this while creating a 'playbook' for communities of practice at Redgate."
 "Postman",explore,tools,true,"<a href='https://www.getpostman.com/'>Postman</a> is an API development environment. Currently being explored by the versioning teams."
 "KeePass",adopt,tools,true,"<a href='https://keepass.info/'>KeePass</a> allows to store all passwords in one encrypted file in GitHub, so you need to care about one master password only, rather than multiple passwords to multiple resources."
+"Semantic UI",explore,languages and frameworks,true,"<a href='https://react.semantic-ui.com/'>Semantic UI</a> is a library of web controls for modern web development, similar to Bootstrap, that has an official React library."
+"honeycomb-semantic-ui-theme",explore,languages and frameworks,true,"An <a href='https://github.com/red-gate/honeycomb-semantic-ui-theme'>NPM package</a> to theme <a href='https://semantic-ui.com/'>Semantic UI</a> in the style of Honeycomb."
```

Comment:
Since we're putting this in Explore, we need a team or two committed to exploring it. The orca team are expecting to start building a cross-platform electron app soon-ish, so we might be one good candidate

Reply:
I've committed to taking a look with Wolves over the next couple of weeks :)  Do we need to name a team in the radar itself?

Reply:
Alternatively we could just put this in Adopt, assuming we've already done enough investigation to say this is what we use now :)

Reply:
> Do we need to name a team in the radar itself?

Looks like we usually do, yeah - probably handy when we want to review the radar and update Explore entries

Reply:
> Alternatively we could just put this in Adopt...
I was of this opinion too, but I think @joselima felt we needed to have an actual team try it first, outside of the timeboxing that was done. I'm for JFDIing and popping it into adopt. THoughts?

Reply:
I'd say let's go for adopt, my only concern is that we don't have a complete set of components yet - it's something we can get done fairly quickly but this was a bit of an experiment. Personally I'd go for Explore for now but if you both think this is past that given all the investigation that's gone into it then I'm happy either way!

Reply:
We were going to test a couple of components out to make sure they could work alongside the current Web Toolkit, so components could be replaced incrementally until it becomes 100% SementicUI. Though also like to JFDing as then we'll find this out anyway.

Reply:
I'd be more comfortable with explore until it's been shown to be working on real work, personally.

Unfortunately I don't think we can commit to doing anything with this soon - we've already just done nearly all our UI work for Teams.